TACC/Core-CMS
# FP-1544: Annotate Lightweight Tags

## Overview

Add script to annotate lightweight tags.

## Related

* [FP-1544](https://jira.tacc.utexas.edu/browse/FP-1544)

## Changes

- Add script.
- Ignore its output (used to manually check if script worked).

## Testing

1. Run the new script.
2. Confirm no errors.
3. Compare the two output files.
4. Confirm the "after" output difference has:
    - two dates per entry
    - no other content is changed

The whitespace in output files is different… sorry 🤷‍♂️ .  Ideas?

## Notes

After this process is approved, only **then** should we actually push the tags to origin. (Otherwise, origin might lose data from incorrect solution.)
